README.adoc

```diff
@@ -119,6 +119,14 @@ can act as starting points or examples:
  provider such as https://stormpath.com/[Stormpath], Active Directory,
  https://developers.google.com/identity/[Google], http://openid.net/[OpenID], etc.
 
+ https://github.com/SMARTRACTECHNOLOGY/devkit-simple-service-example[Create Your Own Microservice]::
```

Comment:
> The categorization of the documentation should be reflected in its presentation.

I agree.

But I also think, that as long as the "Next Steps" are related to the DevKit, they should live here. I can't imagine people enjoy repository-hopping.

By the way, I'm not sure if the current entries in "Next Steps" are valid steps, because they refer to the repositories that are actual parts of the DevKit. There particularly are no guides, but the simple sample project walks you through using the archetype, and extending the DevKit. Thus, it has a guide and therefore should live with the other guides.